node/project-management-app/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "project-management-app",
+  "version": "1.0.0",
+  "description": "A sample project management app for Google Chat",
+  "main": "index.js",
+  "scripts": {
```

Comment:
I don't think so because this app is meant to be deployed as a Cloud Function rather than started locally.

Of course, it's possible to install and start the Cloud Function testing environment so the developer can test a function locally. But I don't think this would be useful for a Chat app, unless using something like ngrok to be able to send Chat events to a function running locally.

Reply:
Got it. Definitely not mandatory to support it for now.